src/ledger/LedgerTxnTrustLineSQL.cpp

```diff
@@ -142,7 +147,7 @@ class BulkUpsertTrustLinesOperation : public DatabaseTypeSpecificOperation<void>
             auto const& tl = le.data.trustLine();
             std::string accountIDStr, issuerStr, assetCodeStr;
             getTrustLineStrings(tl.accountID, tl.asset, accountIDStr, issuerStr,
-                                assetCodeStr);
+                                assetCodeStr, 15);
```

Comment:
If you believe that this is already safe in protocol version 14, then a comment explaining it would be worthwhile. My general opinion here is that you can avoid doing the thinking by doing the thing that is obviously safe, but it's up to you. Same thing about line 318.

Reply:
I ended up using `ledgerVersion`.